ktor-client/ktor-client-features/ktor-client-json/common/src/io/ktor/client/features/json/JsonFeature.kt

```diff
@@ -66,10 +66,20 @@ class JsonFeature internal constructor(
          */
         @KtorExperimentalAPI
         var acceptContentTypes: List<ContentType> = listOf(ContentType.Application.Json)
+            @Deprecated("Use accept() function instead.")
             set(newList) {
                 require(newList.isNotEmpty()) { "At least one content type should be provided to acceptContentTypes" }
                 field = newList
             }
+
+        /**
+         * Adds accepted content types.
+         * It also affects `Accept` request header value.
+         */
+        @KtorExperimentalAPI
+        fun accept(vararg contentTypes: ContentType) {
+            acceptContentTypes = acceptContentTypes + contentTypes
```

Comment:
1. `+=`
2. Deprecation warning should be suppressed here

Reply:
I made `acceptContentTypes` mutable and added the suppression.